src/test/java/uk/gov/companieshouse/mapper/DissolutionResponseMapperTest.java

```diff
@@ -31,4 +40,36 @@ public void mapToDissolutionCreateResponse_generatesLinks() {
         assertEquals("/dissolution-request/12345678", result.getLinks().getSelf());
         assertEquals("/dissolution-request/12345678/payment", result.getLinks().getPayment());
     }
+
+    @Test
+    public void mapToDissolutionGetResponse_mapsToGetResponse() {
+        final Dissolution dissolution = DissolutionFixtures.generateDissolution();
+        final DissolutionGetResponse result = mapper.mapToDissolutionGetResponse(dissolution);
+
+        assertEquals(dissolution.getData().getETag(), result.getETag());
+        assertEquals(DISSOLUTION_KIND, result.getKind());
+        assertEquals("/dissolution-request/12345678", result.getLinks().getSelf());
+        assertEquals("/dissolution-request/12345678/payment", result.getLinks().getPayment());
+        assertEquals(dissolution.getData().getApplication().getStatus(), result.getApplicationStatus());
+        assertEquals(dissolution.getData().getApplication().getReference(), result.getApplicationReference());
+        assertEquals(dissolution.getData().getApplication().getType(), result.getApplicationType());
+        assertEquals(dissolution.getCompany().getName(), result.getCompanyName());
+        assertEquals(dissolution.getCompany().getNumber(), result.getCompanyNumber());
+        assertEquals(Timestamp.valueOf(dissolution.getCreatedBy().getDateTime()), result.getCreatedAt());
+        assertEquals(dissolution.getCreatedBy().getUserId(), result.getCreatedBy());
+        assertEquals(dissolution.getData().getDirectors().get(0).getName(), result.getDirectors().get(0).getName());
+        assertEquals(dissolution.getData().getDirectors().get(0).getEmail(), result.getDirectors().get(0).getEmail());
```

Comment:
Not a fan of these asserts. You are basically writing a mapper all over again. There is no guarantee from looking at the test that none of these fields contain `null` values.

Instead, a unit test should setup the data it is planning to assert on, rather than relying on the data defined in the fixtures. Fixtures can change as they are common. If I change a fixture, I shouldn't have to update other unit tests. Tests should be self contained.

Instead, manually define the values you intend to assert on. If you find that you are setting up too much data for a single test, then that would indicate that breaking it into separate, smaller tests should be considered. Take a look at `DissolutionRequestMapperTest.java` to see what I'm talking about